### ISSUE TITLE
How to add social media links and icons?

### ISSUE THREAD

**Issue by Schokolader** [digitalcraftsman/hugo-creative-theme#17]
How can I add social media links and icons in this theme?

**Comment by Schokolader** [comment]
And how to delete this telephone icon? I deleted the line of the telephone in the config.toml under the section [params.contact] . But I still have the telephone icon.

**Comment by Schokolader** [comment]
ok I got it. It's in the contact.html   But how can I make a link with it? I want to add a Instagram Page. I got the instagram logo instead of the phone icon, but how can I make a link?

**Comment by Schokolader** [comment]
ok I wrote it directly as a html link in the footer.html. Not nice I guess... :)

**Comment by Schokolader** [comment]
Is it possible to add the social media stuff in the config.toml? I guess this would be nice for some first users of this theme... :)

**Comment by digitalcraftsman** [comment]
> How can I add social media links and icons in this theme?

Where would you like to add them?

> ok I got it. It's in the contact.html But how can I make a link with it? I want to add a Instagram Page. I got the instagram logo instead of the phone icon, but how can I make a link?

Add a HTML link like with the email at the right. Perhaps it's better to add a custom variable to the config file under `params`, e.g.

``` toml
[params]
    instagramLink = "https://instagram.com/EXAMPLE"
```

You can access the values in the templates with `{{ .Site.Params.instagramLink }}`. This way you don't need to hard-code it in the template.

> Is it possible to add the social media stuff in the config.toml? I guess this would be nice for some first users of this theme... :)

What exactly do you have in mind? What should go where?

**Comment by Schokolader** [comment]
Yes this is exactly what I thought. And I thought to make this fix in your creative theme template for first users.(only with main social media icons). At first it was hard to find for me where the social media icons are because probably I'm very new to hugo. So I think it would be nice for first users(like I was) to integrate the social media icons in the config.toml. That you only have to fill out your social media links in the config.toml without to write variables in the contact.html.

**Comment by laurabarrera11** [comment]
Hi! I didn't really get how to add social media links, for instance I would like to add my linkedin and github accounts to the website I made using the hugo creative theme? How do I go about doing that? I've tried everything but it never renders it on the screen.
